src/scss/grommet-core/_objects.box.scss

```diff
@@ -63,10 +63,11 @@
   position: relative;
   max-width: 100%;
   width: 100vw;
+  height: 100%;
```

Comment:
How this is different? The PR comment indicates removing "height: 100%", but I still see it here.